### PR TITLE
enable rails assets pipeline integration

### LINKED ISSUE #10: rails asset pipeline integration
currently the js files are just stuffed into the top-level of the gem. Since the gem seems to be targeting rails apps and not just plain ruby, please add the integrations to allow the javascript files to be used via the asset pipeline

see http://guides.rubyonrails.org/asset_pipeline.html#adding-assets-to-your-gems for reference

### PR DESCRIPTION
**Pull request by Jared-Prime**
Following the [Rails guides](http://guides.rubyonrails.org/engines.html#assets) for adding javascript and stylesheet dependencies to gems, I propose adding railties as a dependency and changing the directory structure for assets appropriately. With these changes, when a rails application requiring 'duo_web' runs `bundle exec rake assets:precompile`, the duo javascript and css can be autoloaded 
via [manifest directives](http://guides.rubyonrails.org/asset_pipeline.html#manifest-files-and-directives).

This closes https://github.com/duosecurity/duo_ruby/issues/10

### CODE REVIEW

File: lib/duo_web.rb [hunk 1]
```diff
@@ -1,5 +1,7 @@
 require 'openssl'
 require 'base64'
+require 'rails'
```
Comment (Jared-Prime) [inline lib/duo_web.rb]: this require statement probably not needed

**Comment by AaronAtDuo** [comment]
Closing out this old PR

**Pull request closed without merging**